app/models/hydrus/item.rb

```diff
@@ -897,5 +895,8 @@ def delete
 
 end
 
+Dor::EmbargoMetadataDS.include Hydrus::GenericDS
```

Comment:
I think this would fit better in an initializer or somewhere that it's clear we're doing questionable monkeypatching?